firebase.json

```diff
@@ -29,6 +29,7 @@
       { "source": "/adaptations", "destination": "/resources/platform-adaptations", "type": 301 },
       { "source": "/adaptive*", "destination": "/development/ui/layout/adaptive-responsive", "type": 301 },
       { "source": "/android-release", "destination": "/deployment/android", "type": 301 },
+      { "source": "/android-root-predictive-back", "destination": "https://docs.google.com/document/d/1BGCWy1_LRrXEB6qeqTAKlk-U2CZlKJ5xI97g45U7azk/edit#", "type": 301 },
```

Comment:
Can you prepend `/go` and move it alongside the other design doc go links?

https://github.com/flutter/website/blob/main/firebase.json#L180-L183

Reply:
Whoops sorry, done!